py/README.md

```diff
@@ -1,18 +1,103 @@
-# Python scripts
+# BitBox python scripts
+
+This directory contains scripts to talk to the BitBox device directly via the command line
+ (e.g. `send_message.py`, `load_firmware.py`).
+
+## Setup
+
+These instructions use Python 3, pip, and venv (a lightweight "virtual environment").
+Inside the virtual environment, `python` and `pip` refer to Python 3.
+
+All commands below assume you are in the `py/bitbox02` directory.
+
+### Requirements
+
+- python
+- pip >= 25
 
-**Important: pip >= 25 is required.**  
 Editable installs (`pip install -e …`) are only supported with pip 25 or newer.  
 Older pip versions will fail due to changes in how editable installs are handled (PEP 660).
 
-To use the scripts (e.g. `send_message.py`, `load_firmware.py`):
+### Installing dependencies
+
+Install the required Python dependencies listed in `requirements.txt`:
+
 ```bash
-pip3 install py/bitbox02
+python3 -m venv .venv
```

Comment:
Probably best to put this .venv folder into .gitignore, or place the .venv folder outside of the repo.

Reply:
I checked with `git check-ignore -v .venv` and found out it was already ignored in my user git config
`~/.config/git/ignore`.

I'll update the .gitignore in the root as I think we always want to ignore .venv directories.